spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :system do
+  let(:user) { create(:user) }
+  let(:task) { create(:task, user: user) }
+  let(:another_user) { create(:user, email: 'another_user@example.com', password: 'password', password_confirmation: 'password') }
+  let(:another_task) { create(:task, title: 'another_title', content: 'another_content', user: user) }
+  let(:another_user_task) { create(:task, title: 'another_user_title', content: 'another_user_content', user: another_user) }
+
+  describe 'ログイン前' do
+    it 'タスクの新規作成画面に遷移できないこと' do
+      visit new_task_path
+      expect(page).to have_content 'Login required'
+      expect(page).to have_current_path login_path, ignore_query: true
+    end
+
+    it 'タスクの編集画面に遷移できないこと' do
+      visit edit_task_path(task)
+      expect(page).to have_content 'Login required'
+      expect(page).to have_current_path login_path, ignore_query: true
+    end
+  end
+
+  describe 'ログイン後' do
+    before { login(user) }
+
+    context '入力値が正常な場合' do
+      it 'タスクの新規作成が出来ること' do
+        visit new_task_path
+        fill_in 'Title', with: 'task_title'
+        fill_in 'Content', with: 'task_content'
+        select 'doing', from: 'Status'
```

Comment:
Symbol形式で選択できないか試してみて欲しいです（ダメだったらすみません）
```suggestion
        select :doing, from: 'Status'
```

Reply:
b8cbcf0 にてselect_fomの指定方法をシンボルに変更しました